concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1436,7 +1436,7 @@ Hale-1973-1798-1434	Role_and_Social_Structure-88	hum (a tune)	v	11b1.05	3671	HUM
 Hale-1973-1798-1435	Role_and_Social_Structure-89	beat drum	v	11b1.06		
 Hale-1973-1798-1436	Role_and_Social_Structure-90	play instr.	v	11b1.07	3124	PLAY (INSTRUMENT)
 Hale-1973-1798-1437	Role_and_Social_Structure-91	sound	v	11b1.08	2879	SOUND (OF INSTRUMENT OR VOICE)
-Hale-1973-1798-1438	Role_and_Social_Structure-92	rattle	v	11b1.09		
+Hale-1973-1798-1438	Role_and_Social_Structure-92	rattle	v	11b1.09	300	RATTLE
```

Comment:
Would unmap since this is a nounc in concepticon https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L301